doc/whats_new.rst

```diff
@@ -14,17 +14,34 @@ Current
 Changelog
 ~~~~~~~~~
 
-    - Add `'gamma'` distribution with log link by `Mainak Jas`_.
+    - Add option for fitting `'probit'` distribution by `Ravi Garg`_.
+    - Add option for fitting `'gamma'` distribution with log link by `Mainak Jas`_.
+    - Add support for running :class:`sklearn.model_selection.GridSearchCV` on :class:`pyglmnet.GLM` objects by `Mainak Jas`_.
+    - Improved testing of gradients for optimization by `Mainak Jas`_ and `Pavan Ramkumar`_.
+    - Improved testing of estimated coefficients at convergence by `Mainak Jas`_ and `Pavan Ramkumar`_
+    - Add `n_iter_` attribute to :class:`pyglmnet.GLM` by `Mainak Jas`_, `Olivier Pieters`_, `Peter Foley`_ and `Chris Rogers`_
 
 BUG
 ~~~
 
-    - ...
-
+    - Fixed z cache inconsistency in cdfast when alpha and reg_lambda are nonzero by `Peter Foley`_. 
+    - Fixed incorrect usage of the random_state parameter by `Giovanni De Toni`_.
+    - Fixed incorrect proximal operator for group lasso by `Yu Umegaki`_.
+    - Changed stopping criteria for convergence (a threshold on the change in objective value) which
+      stopped too learly. The new criteria is a threshold on the norm of the gradient, by `Pavan Ramkumar`_. 
+    - Fixed `group` parameter not being passed for Group Lasso by `Beibin Li`_.
+    - Fix URL paths when running on Windows by `Scott Otterson`_.
+    - Made temporary file handling OS independent and usable on a cluster by `Scott Otterson`_.
+    - Replace StratifiedKFold() with KFold(), as StratifiedKFold() doesn't work for continuous values by `Scott Otterson`_.
```

Comment:
hmmm ... wasn't this already in master?

Reply:
He's referring to the examples: https://github.com/glm-tools/pyglmnet/commit/bae902f30638942d08819ec9cd4acdc5f6e33682#diff-3433b590ca8611f80d7687b51a869383